TestTransactionTimeout is flaky

```
--- FAIL: TestTransactionTimeout (0.10s)
    transaction_test.go:291: error code mismatch
         Got: Unknown
        Want: DeadlineExceeded
```

--- a/transaction_test.go
+++ b/transaction_test.go
@@ -325,9 +325,9 @@ func TestTransactionTimeout(t *testing.T) {
 	defer teardown()
 	ctx := context.Background()
 
-	server.TestSpanner.PutExecutionTime(testutil.MethodExecuteStreamingSql, testutil.SimulatedExecutionTime{MinimumExecutionTime: 50 * time.Millisecond})
+	server.TestSpanner.PutExecutionTime(testutil.MethodExecuteStreamingSql, testutil.SimulatedExecutionTime{MinimumExecutionTime: 200 * time.Millisecond})
 	tx, _ := db.BeginTx(ctx, &sql.TxOptions{})
-	if _, err := tx.ExecContext(ctx, "set local transaction_timeout=10ms"); err != nil {
+	if _, err := tx.ExecContext(ctx, "set local transaction_timeout=100ms"); err != nil {
 		t.Fatal(err)
 	}
 	_, err := tx.ExecContext(ctx, testutil.UpdateBarSetFoo)
@@ -367,14 +367,14 @@ func TestTransactionTimeoutSecondStatement(t *testing.T) {
 	ctx := context.Background()
 
 	tx, _ := db.BeginTx(ctx, &sql.TxOptions{})
-	if _, err := tx.ExecContext(ctx, "set local transaction_timeout=40ms"); err != nil {
+	if _, err := tx.ExecContext(ctx, "set local transaction_timeout=200ms"); err != nil {
 		t.Fatal(err)
 	}
 	if _, err := tx.ExecContext(ctx, testutil.UpdateBarSetFoo); err != nil {
 		t.Fatal(err)
 	}
 
-	server.TestSpanner.PutExecutionTime(testutil.MethodExecuteStreamingSql, testutil.SimulatedExecutionTime{MinimumExecutionTime: 60 * time.Millisecond})
+	server.TestSpanner.PutExecutionTime(testutil.MethodExecuteStreamingSql, testutil.SimulatedExecutionTime{MinimumExecutionTime: 300 * time.Millisecond})
 	rows, err := tx.QueryContext(ctx, testutil.SelectFooFromBar, ExecOptions{DirectExecuteQuery: true})
 	if rows != nil {
 		_ = rows.Close()

